sample/shared/compose/src/commonMain/kotlin/com/arkivanov/sample/shared/dialog/DialogContent.kt

```diff
@@ -0,0 +1,33 @@
+package com.arkivanov.sample.shared.dialog
+
+import androidx.compose.material.Text
+import androidx.compose.material.AlertDialog
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+
+@Composable
+fun DialogContent(dialogComponent: DialogComponent?) {
+    dialogComponent?.run {
+        AlertDialog(
+            onDismissRequest = {
+                dialogComponent.onDismissClicked()
```

Comment:
```suggestion
                this.onDismissClicked()
```

Reply:
I'm sorry. Haven't got this. How can we use `this` keyword here?

Reply:
Yes because of the `.run { }`